doc/source/language/flow.rst

```diff
@@ -391,7 +391,10 @@ Note that any ``WAIT`` statement, no matter what the actual expression is, will
     The :ref:`WAIT <wait>` command only causes mainline code
     to be suspended.  Trigger code such as WHEN, ON, LOCK STEERING,
     and LOCK THROTTLE, will continue executing while your program
-    is sitting still on the WAIT command.
+    is sitting still on the WAIT command.  Furthermore, you cannot
```

Comment:
This sounds a bit redundant to me. The part after `Furthermore` pretty much just repeats what the first part states. (Apart from adding 'LOCK WHEELSTEERING' and 'LOCK WHEELTHROTTLE' to the list of locks).